solution.cpp

```diff
@@ -1,23 +1,175 @@
 /* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
- * Released under the [MIT License] (http://opensource.org/licenses/MIT)
- * ------------------------------------------------------------------------- */
+* Copyright &copy; 2015 (Henry Yu) <(henryu1996.csu.fullerton.edu)>
+*Released under the [MIT License] (http://opensource.org/licenses/MIT)
+* ------------------------------------------------------------------------- */
 
 /**
- * TODO: program description
+ * This program converts integers to Roman numerals.
  */
 
 // TODO: `#include`s and function prototypes
+#include <iostream>
+#include <string>
+
+using std::cout;
+using std::cin;
+using std::string;
+using std::endl;
 
 // ----------------------------------------------------------------------------
 
-int main() {
-    // TODO
+#include "stdafx.h"
+#include <iostream>
+using std::cout;
+using std::cin;
+using std::endl;
+
+#include <string>
+using std::string;
+
+int main()
+{
+string roman;
+int integer;
+int part;
+char redo;
+
+while(true)
+{
+    cout << "Please enter an integer: ";
+    cin >> integer;
+
+    if ((integer >= 4000) || (integer <= 0))
+    {
+        cout << "Invalid integer" << endl;
+    }
+    else
```

Comment:
most of this block should have been in a separate function.  no need to change it now though... :-)